API.Microsoft.AspNetCore.Diagnostics.Middleware/NoDocs/Microsoft.AspNetCore.Diagnostics.Latency/RequestCheckpointConstants.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Microsoft.AspNetCore.Diagnostics.Middleware'
+
+namespace Microsoft.AspNetCore.Diagnostics.Latency;
+
+public static class RequestCheckpointConstants
```

Comment:
I don't think these static classes declaring telemetry constants need to be public. I don't understand what the use-case would be. For anyone who wanted to parse the ILogger data after it was emitted or make their own implementation that emits the same values I expect they would just write these same constants into their own code rather than take a ref on this assembly.